legacy/pom.xml

```diff
@@ -51,5 +51,12 @@
             <version>2.2.1</version>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.axonframework.extensions.mongo</groupId>
```

Comment:
I don't like this dependency much since our core repo is now depending on extensions repo. Does it make sense to create a `mongo-legacy` module in `extensions.mongo` repo? If not, we can leave it as is...

Reply:
@abuijze and I wanted to be pragmatic about this, as the token *was* removed from Axon 3, so this original package.
If you feel that strongly about it however, I'll create a new `mongo-legacy` module for that class though.

Reply:
Aha, I guess that package structure for `mongo-legacy` would feel weird `org.axonframework.eventsourcing.eventstore.MongoTrackingToken`. But, IMHO I would search for `mongo-legacy` in `extensions.mongo` repository. Anyway, I'm leaving this decision to you, is not that important after all.